target/linux/malta/image/Makefile

```diff
@@ -45,7 +45,8 @@ endef
 
 define Image/Build
 	$(call Image/Build/$(1))
-	dd if=$(KDIR)/root.$(1) of=$(BIN_DIR)/$(IMG_PREFIX)-root.$(1) bs=128k conv=sync
+	dd if=$(KDIR)/root.$(1) bs=128k conv=sync | \
+		gzip -9n >$(BIN_DIR)/$(IMG_PREFIX)-root.$(1).gz
```

Comment:
gzipping should probably be conditional (CONFIG_TARGET_IMAGES_GZIP), take a look how it's being done in x86 for example

Reply:
I copied this from [armvirt](https://github.com/openwrt/openwrt/blob/d78e229903a3cc42b5117abbe65fb3c5397bd94e/target/linux/armvirt/image/Makefile#L24-L25). Honestly, I see no need to keep a mostly-empty 256MB file unzipped.

Reply:
I do. For local builds it helps to be able to *not* gzip files as it saves one extra step (or intermediate copy) when straightly testing build results in qemu.

Reply:
Ok :smile: Should I make gzipping conditional for both malta and armvirt?